recipes/myproxyclient/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MyProxyClient" %}
+{% set version = "1.4.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "5b7ad3cc3c3cd71595a9b0278a51c138699e229d47a41f2e2add72fbc12f58cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - myproxyclient = myproxy.script:main
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyopenssl
+    - certifi
+
+  run:
+    - python
+    - pyopenssl
+
+test:
+  imports:
+    - myproxy
+    - myproxy.test
+    - myproxy.utils
+  commands:
+    - myproxyclient --help
+
+about:
+  home: https://github.com/cedadev/MyProxyClient
+  license: GNU Library or Lesser General Public License (BSD)
+  summary: 'MyProxy Client'
+
+extra:
+  recipe-maintainers:
+  - agoodman
```

Comment:
Guessing this is probably ok as you guys work together. Correct @agoodman?

To save some unnecessary pings/wait time (we like to check first 😄), you may wish to simply comment in issue ( https://github.com/conda-forge/staged-recipes/issues/139 ) and mention it is ok for @jarifibrahim to include you as a maintainer at his discretion.